src/main/java/edu/kis/powp/jobs2d/command/ICompoundCommand.java

```diff
@@ -12,4 +13,7 @@
 	public default void accept(ICommandVisitor visitor){
 		visitor.visit(this);
 	}
+	public default ICompoundCommand accept(TransformCommandVisitor visitor){
+		return visitor.visit(this);
+	}
```

Comment:
Too bad. I am not sure that this is necessary. TransformCommandVisitor could simply keep the result of visit call as its field (without passing it through accept).